deflate.c

```diff
@@ -202,11 +202,11 @@ int32_t ZNG_CONDEXPORT PREFIX(deflateInit2)(PREFIX3(stream) *strm, int32_t level
 
     strm->msg = NULL;
     if (strm->zalloc == NULL) {
-        strm->zalloc = zng_calloc;
+        strm->zalloc = PREFIX3(calloc);
```

Comment:
This should be exported symbol, so needs updating the list of exported symbols. `PREFIX3` adds `z_` prefix when building in compatibility mode.

Reply:
Why should it be added to export symbols?

Reply:
Nevermind I see zlib.map.